EventFilter/SiStripRawToDigi/python/SiStripDigiToRaw_cfi.py

```diff
@@ -5,6 +5,7 @@
     InputModuleLabel = cms.string('simSiStripDigis'),
     InputDigiLabel = cms.string('ZeroSuppressed'),
```

Comment:
Hi @pieterdavid based on the relval errors across the board:
```
----- Begin Fatal Exception 01-Jun-2018 17:19:10 CEST-----------------------
An exception of category 'Configuration' occurred while
   [0] Constructing the EventProcessor
   [1] Validating configuration of module: class=SiStripDigiToRawModule label='SiStripDigiToRaw'
Exception Message:
Illegal parameters found in configuration.  The parameters are named:
 'InputDigiLabel'
 'InputModuleLabel'
You could be trying to use parameter names that are not
allowed for this plugin or they could be misspelled.
----- End Fatal Exception -------------------------------------------------
```
I think these two:
```
InputModuleLabel = cms.string('simSiStripDigis'),
InputDigiLabel = cms.string('ZeroSuppressed'),
```
should be gone as well to please the descriptions.

Reply:
thanks @mmusich , indeed I forgot to fully update the configs for the packer... I am having a look at that (and running the matrix tests locally before pushing this time ;-) )